resources/puppetlabs/lein-ezbake/staging-templates/project_data.yaml.mustache

```diff
@@ -59,10 +59,10 @@ templates:
 tar_excludes:
   - .gitignore
 build_pe: {{{is-pe-build}}}
-apt_repo_name: {{{repo-name}}}
-apt_nonfinal_repo_name: {{{nonfinal-repo-name}}}
-yum_repo_name: {{{repo-name}}}
-yum_nonfinal_repo_name: {{{nonfinal-repo-name}}}
+repo_name: {{{repo-name}}}
+repo_link_target: 'puppet'
+nonfinal_repo_name: {{{nonfinal-repo-name}}}
+nonfinal_repo_link_target: 'puppet-nightly'
```

Comment:
@melissa I was wondering if we could have these be overwritable with a parameter but maybe default to the hardcoded values?